native/cocos/base/threading/MessageQueue.cpp

```diff
@@ -141,6 +140,10 @@ void MessageQueue::terminateConsumerThread() noexcept {
 
     kick();
     event.wait();
+
+    if (_consumerThread->joinable()) {
+        _consumerThread->join();
```

Comment:
after join, does it need to delete _consumerThread？

Reply:
Yes, need delete it. I will fix it. Thank you.